app/src/main/java/ch/epfl/skysync/components/FlightDetail.kt

```diff
@@ -194,6 +226,33 @@ fun ConfirmFlightTimes(confirmedFlight: ConfirmedFlight, padding: Dp, cardColor:
   }
 }
 
+/**
+ * Composable function for displaying the takeoff and landing times for a finished flight.
+ *
+ * @param finishedFlight The finished flight object containing details to be displayed.
+ * @param padding The padding value to be applied around the content.
+ * @param cardColor The color to be used for the card background.
+ */
+@Composable
+fun FinishedFlightTimes(finishedFlight: FinishedFlight, padding: Dp, cardColor: Color) {
+  val times =
+      mapOf(
+          "Takeoff time" to finishedFlight.takeOffTime,
+          "Landing time" to finishedFlight.landingTime,
+          "Flight duration" to
```

Comment:
Looks a little bit complicated. It is easier to make a `Map<String, String>` like this
```kotlin
  val times =
      mapOf(
          "Takeoff time" to DateUtility.localTimeToString(DateUtility.dateToLocalTime(finishedFlight.takeOffTime)),
          "Landing time" to DateUtility.localTimeToString(DateUtility.dateToLocalTime(finishedFlight.landingTime)),
          "Flight duration" to formatTime(finishedFlight.flightTime))
```
which simplifies `DisaplayMetric`
```kotlin
DisplaySingleMetric(metric = label, value = time)
```

Reply:
Yes